### DISCUSSION TITLE
2020-07-17-02-Text-preprocessing-POS-tagging-and-NER.ipynb

### DISCUSSION THREAD

**Discussion by mgt-strategies** [explosion/spaCy#11363] [Help: Other Questions] [answered]
Can't run this notebook - it doesn't find the data document data file - Abraham Lincoln's Gettysburg Address
Please fix ASAP!

**Comment by ljvmiranda921** [comment, marked as answer]
HI @mgt-strategies , I looked around and [this may be the notebook you're referring to](https://colab.research.google.com/github/goodboychan/chans_jupyter/blob/main/_notebooks/2020-07-17-02-Text-preprocessing-POS-tagging-and-NER.ipynb) ([blogpost](https://goodboychan.github.io/python/datacamp/natural_language_processing/2020/07/17/02-Text-preprocessing-POS-tagging-and-NER.html)). We didn't write this so we recommend contacting the author to find the source of the original file.
